lib/lighthouse/matchers/rspec.rb

```diff
@@ -4,23 +4,40 @@
 require 'lighthouse/matchers'
 require 'lighthouse/audit_service'
 require 'json'
+require 'digest'
+require 'fileutils'
 
 RSpec::Matchers.define :pass_lighthouse_audit do |audit, args = {}|
   score ||= args.fetch(:score, Lighthouse::Matchers.minimum_score)
 
   match do |target|
-    AuditService.new(url(target), audit, score).passing_score?
+    @audit_service = AuditService.new(url(target), audit, score)
+    @audit_service.passing_score?
   end
 
   failure_message do |target|
     <<~FAIL
       expected #{url(target)} to pass Lighthouse #{audit} audit
-      with a minimum score of #{score}
+      with a minimum score of #{score}. Got #{@audit_service.measured_score}.
```

Comment:
🐘 I do sort of prefer the phrasing I used in #26 because it feels grammatically better imo:

```suggestion
      with a minimum score of #{score}, but only scored #{@measured_score}.
```

But I won't block over it :)

(I also can't remember if there was a specific gotcha I hit that meant I had the `to_i`, but I trust you've already considered that)

Reply:
sure